config/development.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         "'self'",
         addonsServerDevCDN,
         analyticsHost,
+        apiDevHost,
```

Comment:
mm why is this needed?

Reply:
I think you answered your own question below. It's needed to enable the `sendBeacon` for the new `/shelves/sponsored/impression/` endpoint when developing locally.